client/data/authorizations/actions.ts

```diff
@@ -18,13 +18,46 @@ import {
 } from 'wcpay/types/authorizations';
 import { STORE_NAME } from '../constants';
 import { ApiError } from 'wcpay/types/errors';
+import { formatCurrency } from 'multi-currency/utils/currency';
 
-const getErrorMessage = ( apiError: {
+interface WCPayError {
 	code?: string;
 	message?: string;
-} ): string => {
+	data?: {
+		status?: number;
+		extra_details?: {
+			minimum_amount?: number;
+			minimum_amount_currency?: string;
+		};
+	};
+}
+
+const getErrorMessage = ( apiError: WCPayError ): string => {
 	// Map specific error codes to user-friendly messages
-	const errorMessages: Record< string, string > = {
+	const getAmountTooSmallError = ( error: WCPayError ): string => {
+		const currency =
+			error.data?.extra_details?.minimum_amount_currency ?? 'USD';
```

Comment:
I feel rather than falling back to these defaults, maybe we could omit the message in case the server doesn't provide the `extra_details`. Or do we see any benefits of this approach?